src/bazel/template-loader.js

```diff
@@ -0,0 +1,30 @@
+var templateUrlRegex = /templateUrl\s*:(\s*['"`](.*?)['"`]\s*([,}]))/gm;
```

Comment:
this file only used with karma, right? seems like there should be a karma subdirectory